tests/ociswrapper/ocis/ocis.go

```diff
@@ -271,3 +199,165 @@ func RunCommand(command string, inputs []string) (int, string) {
 
 	return c.ProcessState.ExitCode(), cmdOutput
 }
+
+func StartService(service string, envMap []string) {
+	// Initialize command args based on service presence
+	cmdArgs := []string{"server"} // Default command args
+
+	if service != "" {
+		cmdArgs = append([]string{service}, cmdArgs...)
+	}
+	// wait for the log scanner to finish
+	var wg sync.WaitGroup
+	wg.Add(2)
+
+	stopSignal = false
+	if retryCount == 0 {
+		defer common.Wg.Done()
+	}
+
+	cmd = exec.Command(config.Get("bin"), cmdArgs...)
+
+	if len(envMap) == 0 {
+		cmd.Env = append(os.Environ(), EnvConfigs...)
+	} else {
+		cmd.Env = append(os.Environ(), envMap...)
+	}
+	log.Println(fmt.Sprintf("%s", cmd.Env))
```

Comment:
```suggestion
	log.Println(cmd.Env)

```
if it's just logging will not this work?

Reply:
`log` is a custom logger and it requires a string parameter

Reply:
This could be done in another PR to scan the error handling and logging consistency.